### PR TITLE
release: release 2.1 version.

### PR DESCRIPTION
**Pull request by moonming**
https://github.com/apache/apisix/milestone/8
2.1 milestone almost finished, it’s time to start release

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -32,6 +33,24 @@
...
+
+### Core
+- :sunrise: **support ENV variable in configuration.** [#2743](https://github.com/apache/apisix/pull/2743)
+- :sunrise: **support TLS connection with etcd.** [#2548](https://github.com/apache/apisix/pull/2548)
+- generate create/update_time automatically. [#2740](https://github.com/apache/apisix/pull/2740)
+- add a deprecate log for enable_websocket in upstream.[#2691](https://github.com/apache/apisix/pull/2691)
+- refactor(consumer): remove id and use username only.[#2679](https://github.com/apache/apisix/pull/2679)
+
+## Plugin
+- :sunrise: **Upgrade protocol to support Apache Skywalking 8.0**[#2389](https://github.com/apache/apisix/pull/2389). So this version only supports skywalking 8.0 protocol.
```
Comment (spacewander) [inline CHANGELOG.md:46]: Should we also mention that skywalking plugin is disabled by default?

Comment (moonming) [reply CHANGELOG.md:46]: ok, I will add to changelog. And we also need to update doc of plugin skywalking.

File: CHANGELOG.md [hunk 2]
```diff
@@ -32,6 +33,24 @@
...
 - [0.6.0](#060)
 
+## 2.1.0
+
+### Core
+- :sunrise: **support ENV variable in configuration.** [#2743](https://github.com/apache/apisix/pull/2743)
+- :sunrise: **support TLS connection with etcd.** [#2548](https://github.com/apache/apisix/pull/2548)
+- generate create/update_time automatically. [#2740](https://github.com/apache/apisix/pull/2740)
+- add a deprecate log for enable_websocket in upstream.[#2691](https://github.com/apache/apisix/pull/2691)
+- refactor(consumer): remove id and use username only.[#2679](https://github.com/apache/apisix/pull/2679)
```
Comment (spacewander) [inline CHANGELOG.md:43]: Should we write it clear that people need to delete the obsolete id from the stored etcd data?

Comment (moonming) [reply CHANGELOG.md]: added

**Comment by moonming** [comment]
I will create a PR to release 2.1, because I created v2.1 branch for it.

**Pull request closed without merging**